src/credentials.rs

```diff
@@ -32,54 +38,78 @@ pub struct Credentials {
 }
 
 impl Credentials {
-    fn location() -> eyre::Result<PathBuf> {
-        let home = home::home_dir().wrap_err("Failed to locate home directory")?;
+    /// Get buffrs config directory.
+    fn buffrs_home() -> Result<PathBuf, VarError> {
+        var("BUFFRS_HOME").map(PathBuf::from)
+    }
 
-        let home = env::var("BUFFRS_HOME").map(PathBuf::from).unwrap_or(home);
+    /// XDG path: uses whichever config directory is appropriate for the platform.
+    ///
+    /// For example, on Linux this might be `~/.config/buffrs/credentials.toml`.
+    fn xdg_path() -> PathBuf {
```

Comment:
I'd call this "config_path" or something similar. Knowledge of the XDG spec shouldn't be required to understand what this is about.